index.html

```diff
@@ -173,14 +176,13 @@ <h3 class="refs__links__title">Hlasování Per Rollam ohledně výroční zpráv
         <p class="refs__links__text">Představenstvo FIT++ schvaluje výroční zprávu FIT++ za rok 2021.</p>
         <a class="refs__links__link" href="/documents/vyrocniZpravy/vz_fitpp_2021_navrh1.pdf"</a><i class="refs__links__link__logo ion ion-ios-cloud-download"></i>Výroční zpráva</a>
         <details class="refs__links__text">
-            <summary>Hashe md5 pro hlasování představenstva</summary>
+            <summary>Členové představenstva</summary>
             <br />
-            <p class="refs__links__text">Byl použit program md5sum (GNU coreutils) 9.0</p>
-            <p class="refs__links__text"><strong>Petr Šťastný:</strong> 2b8e560ec3ebf70fcd986c74a54f2aa8</p>
-            <p class="refs__links__text"><strong>Jakub Horák:</strong> ae47bb3d47a0a31f17d9fe23a28cd489</p>
-            <p class="refs__links__text"><strong>Damián Baraník:</strong> 03889d86abb80fc9db8917f2d1a3e068</p>
-            <p class="refs__links__text"><strong>Jaroslav Kříž:</strong> 1fcb5777ecd89838c9ce588e630ae718</p>
-            <p class="refs__links__text"><strong>Lukáš Hypša:</strong> 42450a2911029e2b0a273f85614664f5</p>
```

Comment:
Nemyslim si, ze na toto by se melo sahat.
Jsou to informace k hlasovani o vyrocni zprave za 2021, tohle nema byt aktualni seznam clenu predstavenstva.
cc @SoptikHa2

Reply:
Už vyřešeno. Akorát neumím koukat